lex/lexer.mll

```diff
@@ -292,13 +292,14 @@ and action = parse
     { action lexbuf }
 
 and skip_char = parse
-  | '\\'? '\010' "'"
+  | '\\'? ('\013'* '\010') "'"
      { incr_loc lexbuf 1;
      }
-  | [^ '\\' '\''] "'" (* regular character *)
+  | [^ '\\' '\'' '\010' '\013'] "'" (* regular character *)
 (* one character and numeric escape sequences *)
   | '\\' _ "'"
   | '\\' ['0'-'9'] ['0'-'9'] ['0'-'9'] "'"
+  | '\\' 'o' ['0'-'7'] ['0'-'7'] ['0'-'7'] "'"
```

Comment:
This is part of #1912 too. I'll remove it there if this gets merged first.

Reply:
I wonder if we should comment that we accept illegal sequences so we can have better error messages later. It might not be guessed by someone reading the code.

Reply:
@314eter, I would expect #1912 to be merged first as It has already been approved.

Reply:
@damiendoligez, do you plan to merge #1912 yourself ?